apps/rush-lib/src/cli/utilities/npm/NpmLinkManager.ts

```diff
@@ -0,0 +1,323 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
>// [](start = 0, length = 2)

To validate that we didn't break any of the NPM behaviors, could you do a behavioral test? 

Run through various operations:
- lazy generate
- regular generate
- install
- clean install
- pull git changes and then do incremental install

Do these operations in separate enlistments using the old and new tool, and diff the log output and common folder contents, just to make sure they are identical.  (But don't spend too much time on it. :-) )